src/entities/supportRequest.ts

```diff
@@ -37,6 +39,12 @@ export class SupportRequest extends BaseEntity {
   @Column({ nullable: false })
   name: string;
 
+  @Column({ nullable: true })
+  primaryLanguage: string;
+
+  @Column('simple-json')
+  extraData: { problemDescription: string };
```

Comment:
So we can use this type in other files we should move it to an interface at the top of this file and export it. Then we can also just use this in the constructor so adding more fields is easier.

```typescript
export const interface SupportRequestExtraData {
  problemDescription: string;
}
```

```suggestion
  extraData: SupportRequestExtraData;
```